cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +95,7 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	utilfeature.DefaultFeatureGate.AddFlag(cmds.PersistentFlags())
```

Comment:
No, not currently. But is there a need for binaries which are relatively external to core like kubeadm to percolate its features?